psp/testing.py

```diff
@@ -25,30 +20,3 @@ def run_click_command(main_func, cmd_args: list[str]):
     assert result.exit_code == 0
 
     return result
-
-
-def make_test_nwp_data_source():
```

Comment:
why have you taken this out? I saw you committed a small zarr file to git. 

In the past we've done this and then regretted it, becasue to adjust that zarr file it looks like a big change in the git history, compared to changing this function

Reply:
I needed some real NWP data because I want the models trained in the tests to use it. If it's random there are no patterns and the data is not used. It was the simplest way to make it work.

I think we shouldn't be afraid to also make synthetic nwp data in tests on top of this if we need it.

Reply:
ah fair enough